libr/bin/p/any.mk

```diff
@@ -8,6 +8,12 @@ ALL_TARGETS+=${TARGET_ANY}
 include $(SHLR)/zip/deps.mk
 
 ${TARGET_ANY}: ${OBJ_ANY}
+ifeq ($(CC),cccl)
```

Comment:
wat. "lib" is implicit by -l you cant link a library that is not prefixed by 'lib' using -l